podcasts/PlaybackManager.swift

```diff
@@ -45,7 +45,7 @@ class PlaybackManager: ServerPlaybackDelegate {
     private let shouldDeactivateSession = AtomicBool()
     private var haveCalledPlayerLoad = false
 
-    private let updateTimerInterval = 1 as TimeInterval
+    private let updateTimerInterval = 0.1 as TimeInterval
```

Comment:
Are we not running the risk of flooding the system with to much notifications? Maybe we should consider a different timer and notification for transcript progress updates.

Reply:
We need to revert that before merging — this is just for testing purposes. Leaving this message so we won't forget.